tgui/packages/tgui/interfaces/CTFPanel.js

```diff
@@ -0,0 +1,64 @@
+import { useBackend } from '../backend';
+import { Box, Button, Section, Flex, Stack } from '../components';
+import { Window } from '../layouts';
+
+export const CTFPanel = (props, context) => {
+  const { act, data } = useBackend(context);
+  const teams = data.teams || [];
+  const enabled = data.enabled || [];
+  return (
+    <Window
+      title="CTF Panel"
+      width={700}
+      height={600}>
+      <Window.Content scrollable>
+        <Section title={enabled} textAlign="center">
+          <Flex wrap="wrap" justify="space-around">
```

Comment:
Small trick that you can use with flex: m={-0.5} on Flex, and m={0.5} on Flex items. This will result in an even gap of `1` between the items.

Which also means you can remove justify, because default stretch would start to look nice.

Reply:
What do you mean by default stretch? Once I make them 50% each, it doesn't look like I really need this anymore.